use/DifferentiationSNP.Rmd

```diff
@@ -120,22 +117,22 @@ genet.dist(Mydata1, method="WC84")
 ## Unsupervised clustering  
 
 We don't know the populations and we are looking for. As recommended by T.
-Jombart, with the function `gpr` we used the maximum  possible number of PCA
+Jombart, with the function `find.clusters()` we used the maximum  possible number of PCA
 axis which is 20 here. See detailed tutorial on this method for more information
 (http://adegenet.r-forge.r-project.org/) In this example, we used
```

Comment:
Shouldn't the link be hyperlinked text instead? Also, the URL as given doesn't actually go to _tutorial on this method_, unlike what the text suggests. It goes to the adegenet website on R-Forge, which says that will soon move to Github (has it meanwhile?). Is the tutorial resource meant to be visited by the reader [this one on R-Forge](http://adegenet.r-forge.r-project.org/documentation.html) (the frame-based structure of the website on R-Forge is terrible for direct linking! cc @thibautjombart), or [this one on Github](https://github.com/thibautjombart/adegenet/wiki/Tutorials), or one of the ones listed on Github (or R-Forge).